flink-runtime/src/test/java/org/apache/flink/runtime/state/SerializationProxiesTest.java

```diff
@@ -126,11 +131,11 @@ public void testKeyedBackendSerializationProxyRoundtripWithSerializerSerializati
 		Assert.assertTrue(serializationProxy.getKeySerializer() instanceof UnloadableDummyTypeSerializer);
 		Assert.assertEquals(keySerializer.snapshotConfiguration(), serializationProxy.getKeySerializerConfigSnapshot());
 
-		for (RegisteredKeyedBackendStateMetaInfo.Snapshot<?, ?> meta : serializationProxy.getStateMetaInfoSnapshots()) {
-			Assert.assertTrue(meta.getNamespaceSerializer() instanceof UnloadableDummyTypeSerializer);
-			Assert.assertTrue(meta.getStateSerializer() instanceof UnloadableDummyTypeSerializer);
-			Assert.assertEquals(namespaceSerializer.snapshotConfiguration(), meta.getNamespaceSerializerConfigSnapshot());
-			Assert.assertEquals(stateSerializer.snapshotConfiguration(), meta.getStateSerializerConfigSnapshot());
+		for (StateMetaInfoSnapshot meta : serializationProxy.getStateMetaInfoSnapshots()) {
```

Comment:
Change like
`RegisteredKeyedBackendStateMetaInfo.Snapshot<?, ?> meta -> RegisteredKeyedBackendStateMetaInfo(StateMetaInfoSnapshot)`
and then using original assertions might be more complete version of test.
Also for other meta wrappers in further tests

Reply:
👍 Good point